### DISCUSSION TITLE
Full date-time in `ls` command

### DISCUSSION THREAD

**Discussion by RagibHasin** [nushell/nushell#13569] [Q&A] [answered]
Humanized date-time is the right thing for most of the cases. But how can I know the exact time of creation of a file?

**Comment by pewill** [comment, marked as answer]
You can use `format date` similar to this: 
`ls -l foo.txt | update created {format date}`

**Reply by fdncred** [reply]
Agreed. I use this sometimes because it's verbose a bit more orderly.
```nushell
ls | update modified {format date %+}
╭─#──┬────────name─────────┬─type─┬───size───┬──────────────modified───────────────╮
│ 0  │ CITATION.cff        │ file │    812 B │ 2024-06-07T08:51:12.251176250-05:00 │
│ 1  │ CODE_OF_CONDUCT.md  │ file │   3.4 KB │ 2023-03-24T16:44:17.865147299-05:00 │
│ 2  │ CONTRIBUTING.md     │ file │  11.2 KB │ 2024-06-05T07:14:19.035217639-05:00 │
│ 3  │ Cargo.lock          │ file │ 176.0 KB │ 2024-08-07T16:15:14.026136531-05:00 │
│ 4  │ Cargo.toml          │ file │   9.1 KB │ 2024-08-07T16:15:14.026606281-05:00 │
```

**Reply by RagibHasin** [reply]
@fdncred what table theme is it? There does not seem to be much visualized documentation for table themes anywhere.

**Reply by fdncred** [reply]
table `mode = rounded`, `header_on_separator = true`. these settings are in the config.nu file. theme is called mode in the config file. we should straighten that out eventually.

here's a one-liner to look at the themes/modes
```nushell
table --list | each {|r| print ($r); print (ls | first 3 | table --theme $r)}
```
